packages/react-ui-components/src/SelectBox_Header/selectBox_Header.tsx

```diff
@@ -82,6 +82,8 @@ export default class SelectBox_Header extends PureComponent<SelectBox_Header_Pro
 
         return (
             <span>
+                // TODO: themr looses default props type information of components
```

Comment:
TODO: When exporting the component enhanced with themr, just reset the type to the original component type with `export default themr(...) as Component`.